Работа над дипломным проектом. Задание 2 – Api

mchvell opened the pull request:
Добавлено:
- структура проекта
- конфиг с конфигурацией урла и jvt
- тесты
- 2 хелпера-генератора для получения случайных бургеров и создания случайных юзеров
- отчеты о прохождение тестов

Commit:
Работа над дипломным проектом. Задание 2 – Api

Добавлено:
- структура проекта
- конфиг с конфигурацией урла и jvt
- тесты
- 2 хелпера-генератора для получения случайных бургеров и создания случайных юзеров

AlexKitIt commented on config.py:
```diff
@@ -0,0 +1,4 @@
+class BaseConfig:
+    base_url = "https://stellarburgers.nomoreparties.site"
+
+    authorization = "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY2YjBlZTNhOWVkMjgwMDAxYjQ4NTJmYiIsImlhdCI6MTcyNzE1NDU4OSwiZXhwIjoxNzI3MTU1Nzg5fQ.3o8RBHWm4LB85SzN_07WbIucYi8UhK_YC7iW80Rqk18"
```
Нужно исправить: сенсетив данные не могу храниться в проекте

mchvell replied:
Исправлено

AlexKitIt commented on tests/test_change_user_profile.py:
```diff
@@ -0,0 +1,29 @@
+class TestChangeUserProfile:
+
+    @allure.description("Изменяем имя для существующего юзера с токеном авторизации")
+    def test_change_name_for_existing_user_with_auth_token(self):
```
Нужно исправить: здесь и далее: над каждым тестовым методом ставится аннотация allure.title

mchvell replied:
Исправлено

Commit:
Работа над Дипломным проектом. API тестирование

Исправлено:
- Из проекта удален jwt токен, как константа
- Добавлено 2 фикстуры – получение jwt для существующего юзера, создание юзера и получение его jwt
